examples/tldraw-example/package.json

```diff
@@ -24,10 +24,10 @@
     "@types/react-router-dom": "^5.1.8",
     "concurrently": "6.0.1",
     "create-serve": "1.0.1",
-    "dotenv": "^10.0.0",
     "esbuild": "^0.13.8",
     "esbuild-envfile-plugin": "^1.0.1",
     "esbuild-serve": "^1.0.1",
+    "firebase": "^9.6.1",
```

Comment:
This would be a major dependency. Steve needs to confirm.

Reply:
Ah yeah this is just for the example, the package itself does not need it